pkg/systemd/superuser-alert.jsx

```diff
@@ -33,15 +33,13 @@ export class SuperuserAlert extends React.Component {
         super();
         this.superuser = cockpit.dbus(null, { bus: "internal" }).proxy("cockpit.Superuser", "/superuser");
         this.superuser.addEventListener("changed", () => { this.setState({}) });
-        this.state = { dismissed: false, show_unlock_dialog: false };
```

Comment:
Why would we not make the alert dismissable? If someone really does not want to see this any more, I don't see a reason to force this message to stay around. You can always use the "Limited" indicator to go back to sudo.